extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -238,15 +237,19 @@ export class DraggableDrawer extends AMP.BaseElement {
    * @private
    */
   startListeningForTouchEvents_() {
-    // If the element is a direct descendant of amp-story-page, authorize
-    // swiping up by listening to events at the page level. Otherwise, only
-    // authorize swiping down to close by listening to events at the current
-    // element level.
+    // If the element is a direct descendant of amp-story-page or a decendent
+    // of amp-story-shopping-attachment, authorize swiping up by listening to
+    // events at the page level. Otherwise, only authorize swiping down to
+    // close by listening to events at the current element level.
     const parentEl = this.element.parentElement;
-    const el = dev().assertElement(
+    let el = dev().assertElement(
       parentEl.tagName === 'AMP-STORY-PAGE' ? parentEl : this.element
     );
 
+    if (parentEl.tagName === 'AMP-STORY-SHOPPING-ATTACHMENT') {
+      el = dev().assertElement(this.element.closest('amp-story-page'));
+    }
```

Comment:
In the case where the parent is the shopping attachment, this is currently setting `el` to `this.element` and then setting it immediately afterwards to `this.element.closest('amp-story-page')`. I think we can prevent this by using an if-else statement. Maybe something like this:
```
let el;
if (parentEl.tagName === 'AMP-STORY-PAGE') {
  el = parentEl;
} else if (parentEl.tagName === 'AMP-STORY-SHOPPING-ATTACHMENT') {
  el = this.element.closest('amp-story-page');
} else {
  el = this.element;
}
dev().assertElement(el);
```